app/components/template-viewer/template-viewer.sass

```diff
@@ -10,3 +10,6 @@
     font-family: $font-family-sans-serif
     margin-bottom: $sbh-margin-bottom
 
+.template-item
+  &:hover
+    background-color: $temp-select-color
```

Comment:
You should install this so it automatically adds a newline to end of file and sets up the correct indentation/etc. That way @develra and I can quit bugging you. 

https://github.com/sindresorhus/editorconfig-sublime